content/docs/command-reference/exp/branch.md

```diff
@@ -15,17 +15,19 @@ positional arguments:
 
 ## Description
 
-Makes a
-[Git branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
-off the last commit (`HEAD`) based on the given `experiment`, using the `branch`
-name provided. This makes the given experiment
-[persistent](/doc/user-guide/experiment-management#persistent-experiments) in
-the repo.
-
-In most cases this is similar to using `dvc exp apply` (applies the `experiment`
-results to the workspace) followed by Git branching and committing, except that
-`dvc exp branch` **does not** switch into the created `branch`. Equivalent to
-this:
+Makes a named Git
+[`branch`](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
+containing the target `experiment` (making it
+[persistent](/doc/user-guide/experiment-management#persistent-experiments)). The
+branch is based on the commit that was active (`HEAD`) at the time that the
+target experiment was run.
+
+This command is useful, for example, in cases where `dvc exp apply` fails due to
+significant diversion between the project version at the time of the experiment
+run and the current <abbr>workspace</abbr>.
+
+Note that `dvc exp branch` **does not** switch into the new branch created. It's
+equivalent to:
```

Comment:
My vote is still to drop this since it's slightly inaccurate, and I don't think it adds much that's not already covered by the description, but I know we discussed it in a previous PR, so 🤷 .

Reply:
Hmmm yeah. I think you're right. ✖️'d